bindings/python/unicorn/unicorn.py

```diff
@@ -262,6 +263,14 @@ def mem_protect(self, address, size, perms=UC_PROT_ALL):
         if status != UC_ERR_OK:
             raise UcError(status)
 
+    # return CPU mode at runtime
+    def query(self, query_mode=UC_QUERY_MODE):
```

Comment:
please dont set default value for query_mode here, because this is supposed to be used for other type of queries in the future.